### DISCUSSION TITLE
Extra Voting Rewards Issue

### DISCUSSION THREAD

**Discussion by FestusDM** [Zrips/CMI#4275] [Q&A]
**Question: **
Players have reported that they are not getting the extra voting rewards due to full inventory. Is it supposed to give players the rewards anyways, even if they got a full inventory?

**Comment by CloudeLecaw** [comment]
I think no, finally the reward is a command, when inventory is full it get response that inventory was full or it drops item infront of them. There is no quene for this. A solution would may be to give them access to a special kit an the access gets removed when they take the kit. You could design a customalias that give them all kits where they have access to. Also its possible to add onetime kits that stucks in the amount of useage if they take it once. So if they vote for 6 sites they can take the kit also 6 times.
Well this is just a suggestion but does not prevent them from not getting a reward item if the inventory is full allready.

**Comment by mrfloris** [comment]
To be honest, if the inventory is full it should drop on the floor. if a /give doesn't do that, maybe put the item in a private kit and run the kit command as a reward, it will then drop on the floor if the inventory is full.

**Comment by CloudeLecaw** [comment]
Si next step would be to wait until Player is online (msans its stored somewhere) and give on his next login. Where it should be droped else if the player is offline with full invnentory? ;)

**Reply by mrfloris** [reply]
Going out of scope of just handling caught votes is more something for a full fledge voting plugin. Like VotingPlugin, but yes, if some more features could come to handle the basics, i would not mind (personally)
